### PR TITLE
Zoom label

### PR DESCRIPTION
**Pull request by Rdornier**
Adding the zoom level in the label list

### CODE REVIEW

File: omero_figure/scripts/omero/figure_scripts/Figure_To_Pdf.py [hunk 1]
```diff
@@ -1299,6 +1299,8 @@ def draw_labels(self, panel, page):
                             label_value.append(channel["label"])
                     label_value = " ".join(label_value)
 
+                elif  prop_nf[0] in ["zoom"]:
```
Comment (will-moore) [inline omero_figure/scripts/omero/figure_scripts/Figure_To_Pdf.py:1302]: The build is failing flake8 checks due to this line
```
flake8.main.application   MainProcess   1787 INFO     Found a total of 79 violations and reported 1
./omero_figure/scripts/omero/figure_scripts/Figure_To_Pdf.py:1302:21: E271 multiple spaces after keyword
```

Comment (Rdornier) [reply omero_figure/scripts/omero/figure_scripts/Figure_To_Pdf.py]: Thanks, I suppressed the extra space, it should be ok now

File: omero_figure/templates/figure/index.html [hunk 2]
```diff
@@ -605,6 +605,11 @@ <h4>Dynamic properties</h4>
                             <td>Dataset ID: [dataset.id]</td>
                             <td>Dataset ID: 1654</td>
                         </tr>
+                        <tr>
+                            <th>Zoom</th>
+                            <td>[zoom] %</td>
```
Comment (will-moore) [inline omero_figure/templates/figure/index.html:610]: The ` %` is generated by the `[zoom]` itself, so you have `[zoom] %` you actually get `[zoom] % %`.
So I think you can just remove the ` %` from this line.

Comment (Rdornier) [reply omero_figure/templates/figure/index.html:610]: It takes me moment but now I get it. Yes, we can remove the `` %`` from the tips.

**Comment by will-moore** [comment]
Thanks - merging...

**Pull request merged**